api-reference/receipts/get-started.markdown

```diff
@@ -7,21 +7,32 @@ layout: reference
 
 ## Get Started
 
+- [Overview](#overview)
 - [Version](#version)
 - [Regional Availability](#regional-availability)
 - [Explore the API](#explore-the-api)
   - [Prerequisites](#prerequisites)
-  - [Endpoints](#endpoints)
-    - [Acquire an Access Token](#retrieve-a-user-access-token)
-    - [GET Service Index](#endpoint-service-index)
-    - [GET Schemas](#endpoint-schemas)
-    - [POST Receipts](#endpoint-post-a-receipt)
-    - [GET Receipts by User ID](#endpoint-get-receipts-by-userid)
-    - [GET Receipts by Receipt ID](#endpoint-get-a-receipt-by-id)
-    - [GET Receipt Image by Receipt ID](#endpoint-get-receipt-image)
-    - [GET Receipt Status by Receipt ID](#endpoint-get-receipt-status)
+  - [Acquire an Access Token](#retrieve-a-user-access-token)
   - [Explore the API in JavaScript](#explore-the-api-in-javascript)
 
+### Overview
+
+The Receipts V4 API aims to make our service as useful as possible. In order to do so we accept three different formulae for posting a receipt:
```

Comment:
I think the first sentence of the overview is of dubious usefulness. If we take it out then the second sentence can be shortened and still do a good job of introducing the three receipt types.

Reply:
✅